lib/ui5Framework/AbstractResolver.js

```diff
@@ -43,7 +43,11 @@ class AbstractResolver {
 			throw new Error(`AbstractResolver: Missing parameter "version"`);
 		}
 
-		this._ui5HomeDir = ui5HomeDir ? path.resolve(ui5HomeDir) : path.join(os.homedir(), ".ui5");
+		this._ui5HomeDir = ui5HomeDir ?
```

Comment:
This assignment can be simplified a lot, as path.resolve should always be called now.